_studio/shared/umc/codec/h264_dec/src/umc_h264_dec_defs_yuv.cpp

```diff
@@ -66,15 +66,19 @@ void H264DecYUVBufferPadded::deallocate()
 
     m_pYPlane = m_pUPlane = m_pVPlane = m_pUVPlane = 0;
 
-    m_lumaSize.width = 0;
-    m_lumaSize.height = 0;
+    m_lumaSize = {0, 0};
     m_pitch_luma = 0;
     m_pitch_chroma = 0;
 }
 
 void H264DecYUVBufferPadded::Init(const VideoDataInfo *info)
 {
     VM_ASSERT(info);
+    if (info == nullptr)
+        throw h264_exception(UMC_ERR_NULL_PTR);
+    VM_ASSERT(info->GetNumPlanes());
+    if (info->GetNumPlanes() == 0)
+        throw h264_exception(UMC_ERR_NULL_PTR);
```

Comment:
not sure if NULL_PTR correct status here. you are not checking pointer actually

Reply:
yes, but check (line 80) avoid possible referencing from GetPlaneInfo(0)

Reply:
why do we need asserts if we check conditions next line?